src/arena/Sprint.js

```diff
@@ -471,31 +483,41 @@ function Sprint(props) {
                     {props.redux.sprint[activeSprintId].teams[
                       index
                     ].missions.map((mission, idx) => (
-                      <Button
+                      <Chip
+                        label={`Day ${idx + 1}`}
+                        variant="options"
+                        status={displayDay === idx ? "selected" : null}
                         onClick={() => {
                           setDisplayDay(idx);
                         }}
                         // eslint-disable-next-line react/no-array-index-key
                         key={idx}
-                      >
-                        Day {idx + 1}
-                      </Button>
+                      />
                     ))}
                   </div>
                   <div className="flex">
-                    {props.user.admin === true ? (
+                    {props.user.admin === false ? (
                       <div>
                         <p>Status: {status}</p>
-                        <div className="flex-apart">
-                          <Button onClick={() => setStatus("early")}>
-                            Set to Early
-                          </Button>
-                          <Button onClick={() => setStatus("active")}>
-                            Set to Active
-                          </Button>
-                          <Button onClick={() => setStatus("inactive")}>
-                            Set to Inactive
-                          </Button>
+                        <div>
```

Comment:
Great idea to use the `Chip` element! Better than using Buttons. Is the `div` surrounding the chips necessary? And the `div` surrounding the <p> tag and the Chips - can you use React.Fragments instead? They offer better performance, you can learn more here! https://www.sitepoint.com/react-fragments-introduction/

Reply:
As you see random divs that don't seem to have associated styling with them, feel free to replace them with <React.Fragment> or <> elements so we can remove unnecessary divs from the repository!

Reply:
Will do 🙂

Reply:
Great! Just request 're-review' once you are done :) By the way, we have a contributor only Discord channel. We only have a few people, but if you would like to join - leave your email here and I will send you an invite.

Reply:
I will be happy to join :) 

ananas.nasya@gmail.com